test/test_trixi2img.jl

```diff
@@ -4,7 +4,7 @@ using Trixi
 using Trixi2Img
 
 # pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
-const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
+const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "2d")
```

Comment:
You also need to update the lower bound on Trixi.jl in `test/Project.toml` to `v0.2.0` for this to work.

Reply:
Yes, I am waiting here for (hopefully) you to fix #15

Reply:
Done

Reply:
Will you also merge #16?

Reply:
I've made a necessary modification there. I will merge it when the tests pass - and you don't beat me with that :wink:

Reply:
Merged